File: Date.py
def preprocess(date):
    date = date.replace(",", " ")
    list = date.split(" ")
    list = [x for x in list if x]
    return list[0], list[1], list[2]


def calcuate_day(day, month, year):
    month_dict = {"jan" : 0, "feb" : 31, "mar" : 28 + month_dict["feb"]}


date = "30 March 2019"
day, month, year = preprocess(date)
print(calcuate_day(day, month, year))

File: Date.java
public class Date {
	public static boolean is_ambiguous(String date) {
		String[] date_list = date.split("/");
		return Integer.parseInt(date_list[0]) <= 12 && Integer.parseInt(date_list[1]) <= 12;
	}
	
	public static void main(String args[]) {
		String date = new String("01/02/1999");
		System.out.println(is_ambiguous(date));
	}
}
